client/app/common/less/popover.less

```diff
@@ -81,6 +81,12 @@
                         color: #999;
                     }
 
+                    .large-popover-item {
```

Comment:
Please use helper class instead: `class="border-left-0 border-right-0 w-100"`

Reply:
this is the equivalent of inline css, I really think it's not in our interest